### PR TITLE
Document to install K8s on CentOS on Vagrant 

### LINKED ISSUE #1102: Create document for K8s on CentOS for vagrant
Choose one: FEATURE REQUEST

**What happened**: K8s intallation on CentOS

**What you expected to happen**: Document the installation steps

### PR DESCRIPTION
**Pull request by mahebbar**
**What this PR does / why we need it**: Steps to install K8s on CentOS

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #1102

### CODE REVIEW

File: documentation/source/install/on_premise_solutions.rst [hunk 1]
```diff
@@ -96,6 +96,301 @@ The *ubuntu@kubemaster-01:~$ kubectl apply -f demo/jupyter/demo-jupyter-openebs.
 * Create an OpenEBS Volume and mount to the Jupyter Server Pod (/mnt/data) (kubectl get pvc) (kubectl get pv) (kubectl get pods)
 * Expose the Jupyter Server to external world through http://NodeIP:8888 (NodeIP is any of the nodes' external IP) (kubectl get pods)
 
+Installing Kubernetes on CentOS in Vagrant VMs
+----------------------------------------------------
+The following procedure helps you install Kubernetes on CentOS version 7.4 and use OpenEBS on a local machine. You will be setting up a 3 node cluster comprising of 1 Master and 2 Worker Nodes running Kubernetes 1.8.5.
```
Comment (yudaykiran) [inline documentation/source/install/on_premise_solutions.rst]: OpenEBS on a local machine = OpenEBS on that cluster.

File: documentation/source/install/on_premise_solutions.rst [hunk 2]
```diff
@@ -96,6 +96,301 @@ The *ubuntu@kubemaster-01:~$ kubectl apply -f demo/jupyter/demo-jupyter-openebs.
...
+          vmCfg.vm.network "private_network", ip: "172.28.128.32"
+        end
+
+        config.vm.define "worker-02" do |vmCfg|
+          vmCfg.vm.hostname = "worker-02"
+          vmCfg.vm.network "private_network", ip: "172.28.128.33"
+        end
+    end
+
+Verifying the Vagramt VMs State
```
Comment (yudaykiran) [inline documentation/source/install/on_premise_solutions.rst]: Vagramt = Vagrant

File: documentation/source/install/on_premise_solutions.rst (hunk 1 above)
Comment (mahebbar) [reply documentation/source/install/on_premise_solutions.rst]: Done!

File: documentation/source/install/on_premise_solutions.rst (hunk 2 above)
Comment (mahebbar) [reply documentation/source/install/on_premise_solutions.rst]: Thanks! :( Done!

**Pull request merged**